test/convexhull.jl

```diff
@@ -0,0 +1,7 @@
+@testset "upperhull" begin
```

Comment:
The name of this test set is `Convex Hull`, upper hull is just a piece of the solution, right?